==251999== Memcheck, a memory error detector
==251999== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==251999== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==251999== Command: ./puny python3_scripts/audio_test.py python3_scripts/autoencoder.py python3_scripts/bipartite.py python3_scripts/chem.py python3_scripts/classification.py python3_scripts/coins.py python3_scripts/data_manipulation.py python3_scripts/empty.py python3_scripts/fib.py python3_scripts/func.py python3_scripts/get_accuracies.py python3_scripts/get_data.py python3_scripts/graph.py python3_scripts/heap.py python3_scripts/image.py python3_scripts/images_and_metadata.py python3_scripts/insert.py python3_scripts/integration.py python3_scripts/iris.py python3_scripts/levenshtein.py python3_scripts/linear_regression.py python3_scripts/mcounter.py python3_scripts/MelSpecPlayground.py python3_scripts/merge.py python3_scripts/mnist.py python3_scripts/multiply.py python3_scripts/perceptron.py python3_scripts/prob1.py python3_scripts/problem2.py python3_scripts/__pycache__ python3_scripts/quick.py python3_scripts/script.py python3_scripts/sort.py python3_scripts/stats.py python3_scripts/subsequence.py python3_scripts/test.py python3_scripts/train_test_val.py python3_scripts/tree.py python3_scripts/util.py
==251999== 
Skipping python3_scripts/__pycache__. Not a .py file or does not exist
==251999== 
==251999== HEAP SUMMARY:
==251999==     in use at exit: 32,842 bytes in 3 blocks
==251999==   total heap usage: 65,451 allocs, 65,448 frees, 2,007,395 bytes allocated
==251999== 
==251999== 8 bytes in 1 blocks are still reachable in loss record 1 of 3
==251999==    at 0x483B7F3: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==251999==    by 0x10DDF9: yyalloc (lex.yy.c:2492)
==251999==    by 0x10D8C2: yyensure_buffer_stack (lex.yy.c:2190)
==251999==    by 0x1095C3: real_yylex (lex.yy.c:902)
==251999==    by 0x10DEF4: yylex (punylex.l:246)
==251999==    by 0x10F22F: main (main.c:52)
==251999== 
==251999== 64 bytes in 1 blocks are still reachable in loss record 2 of 3
==251999==    at 0x483B7F3: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==251999==    by 0x10DDF9: yyalloc (lex.yy.c:2492)
==251999==    by 0x10D437: yy_create_buffer (lex.yy.c:2030)
==251999==    by 0x1095ED: real_yylex (lex.yy.c:904)
==251999==    by 0x10DEF4: yylex (punylex.l:246)
==251999==    by 0x10F22F: main (main.c:52)
==251999== 
==251999== 32,770 bytes in 1 blocks are still reachable in loss record 3 of 3
==251999==    at 0x483DFAF: realloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==251999==    by 0x10DE22: yyrealloc (lex.yy.c:2505)
==251999==    by 0x10CB71: yy_get_next_buffer (lex.yy.c:1754)
==251999==    by 0x10C846: real_yylex (lex.yy.c:1616)
==251999==    by 0x10DEF4: yylex (punylex.l:246)
==251999==    by 0x10F22F: main (main.c:52)
==251999== 
==251999== LEAK SUMMARY:
==251999==    definitely lost: 0 bytes in 0 blocks
==251999==    indirectly lost: 0 bytes in 0 blocks
==251999==      possibly lost: 0 bytes in 0 blocks
==251999==    still reachable: 32,842 bytes in 3 blocks
==251999==         suppressed: 0 bytes in 0 blocks
==251999== 
==251999== For lists of detected and suppressed errors, rerun with: -s
==251999== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
